staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1/types.go

```diff
@@ -186,3 +186,346 @@ type WebhookMatchCondition struct {
 	// Documentation on CEL: https://kubernetes.io/docs/reference/using-api/cel/
 	Expression string `json:"expression"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// AuthenticationConfiguration provides versioned configuration for authentication.
+type AuthenticationConfiguration struct {
```

Comment:
the code in v1beta1 was added in:
- https://github.com/kubernetes/kubernetes/pull/123696/files
3 PRs that touched the code afterwards:
- https://github.com/kubernetes/kubernetes/commit/916c7867f7ea766824728851a25b01ebbc600491
- https://github.com/kubernetes/kubernetes/commit/121607e80963370c1838f9f620c2b8552041abfc
- https://github.com/kubernetes/kubernetes/commit/5e6a4937f5a3e20dd77238946220461332ecddff

I do not see any version specific manual defaults or conversions. Will take one more pass in a bit

Reply:
defaults and conversions look identical